modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/api/construction/ConstructionPreprocess.scala

```diff
@@ -0,0 +1,24 @@
+package org.tessellation.rosetta.domain.api.construction
+
+import org.tessellation.ext.derevo.magnoliaCustomizable.snakeCaseConfiguration
+import org.tessellation.rosetta.domain._
+import org.tessellation.rosetta.domain.amount.Amount
+import org.tessellation.rosetta.domain.operation.Operation
+
+import derevo.circe.magnolia.{customizableDecoder, customizableEncoder}
+import derevo.derive
+
+case object ConstructionPreprocess {
+  @derive(customizableDecoder)
+  case class Request(
+    networkIdentifier: NetworkIdentifier,
+    operations: List[Operation],
+    maxFee: Option[List[Amount]],
+    suggestedFeeMultiplier: Option[Double]
```

Comment:
can you show me these properties in the rosetta specification? I cannot find any on the preprocess request: 
https://www.rosetta-api.org/docs/models/ConstructionPreprocessRequest.html

Reply:
I just copied the class over and didn't think to consult the specification. That's on me. Removed.